fossunited/foss_hackathon/doctype/foss_hackathon_team/test_foss_hackathon_team.py

```diff
@@ -1,60 +1,55 @@
-from datetime import datetime, timedelta
-
 import frappe
-from faker import Faker
 from frappe.tests import IntegrationTestCase
 
-from fossunited.doctype_ids import HACKATHON, HACKATHON_PARTICIPANT, HACKATHON_TEAM
+from fossunited.doctype_ids import HACKATHON_TEAM_MEMBER
+from fossunited.tests.utils import (
+    insert_test_hackathon,
+    insert_test_hackathon_participant,
+    insert_test_hackathon_team,
+)
 
 
 class TestFOSSHackathonTeam(IntegrationTestCase):
     def setUp(self):
-        self.faker = Faker()
-        self.hackathon = frappe.get_doc(
-            {
-                "doctype": HACKATHON,
-                "hackathon_name": self.faker.name(),
-                "permalink": self.faker.slug(),
-                "start_date": datetime.now(),
-                "end_date": datetime.now() + timedelta(days=1),
-                "hackathon_description": self.faker.text(),
-                "max_team_size": 2,
-            }
-        ).insert()
-
-        self.team = frappe.get_doc(
-            {
-                "doctype": HACKATHON_TEAM,
-                "team_name": self.faker.name(),
-                "hackathon": self.hackathon.name,
-            }
-        ).insert()
+        self.hackathon = insert_test_hackathon(is_team_mandatory=True, max_team_members=3)
+
+        self.team = insert_test_hackathon_team(hackathon=self.hackathon)
 
         self.participants = []
-        for _ in range(3):  # since max_team_size is 2, we need 3 participants to test the code
-            participant = frappe.get_doc(
-                {
-                    "doctype": HACKATHON_PARTICIPANT,
-                    "full_name": self.faker.name(),
-                    "email": self.faker.email(),
-                    "hackathon": self.hackathon.name,
-                }
-            ).insert()
+        for _ in range(4):  # since max_team_size is 2, we need 3 participants to test the code
```

Comment:
but `max_team_members` is 3 in line 14 and i'm not sure if `max_team_size` is the same as that and you say 3 participants but we iterate 4 times

Reply:
Fixed this. I refactored tests but forgot to update this line. We require 4 participants overall to test exceeding member case